### PR TITLE
fix(material-experimental/mdc-tooltip): make background color opaque

### PR DESCRIPTION
**Pull request by wagnermaciel**
* due to a11y concerns surrounding the 0.6 background color opacity of the
  mdc-tooltip, we are switching to using our own custom background color
  instead of the one defined in the MDC styles.

### CODE REVIEW

File: src/material-experimental/mdc-tooltip/tooltip.scss [hunk 1]
```diff
@@ -11,4 +14,8 @@
   // depending on the state of the overlay. For tooltips the overlay panel should never enable
   // pointer events. To overwrite the inline CSS from the overlay reference `!important` is needed.
   pointer-events: none !important;
+
+  @include tooltip-theme.fill-color(
+    map.get(palette.$light-theme-background-palette, tooltip)
```
Comment (crisbeto) [inline src/material-experimental/mdc-tooltip/tooltip.scss:19]: We should have a comment about why it's done this way. Also shouldn't these styles be in the tooltip theme?

**Comment by wagnermaciel** [comment]
#### Original `mat-tooltip`

Screen Shot 2021-07-02 at 9 52 57 AM

#### Old `mat-mdc-tooltip`

Screen Shot 2021-07-02 at 9 49 51 AM

#### New `mat-mdc-tooltip`

Screen Shot 2021-07-02 at 11 42 50 AM

File: src/material-experimental/mdc-tooltip/tooltip.scss (hunk 1 above)
Comment (wagnermaciel) [reply src/material-experimental/mdc-tooltip/tooltip.scss:19]: I thought we kept all of the custom styles inside the `<component>.scss`, no?

Comment (crisbeto) [reply src/material-experimental/mdc-tooltip/tooltip.scss:19]: No, we can have custom styles anywhere, but the properties based on the theme (e.g. colors) need to be in the theme file.

**Review by jelbourn — APPROVED** [review]
LGTM

**Pull request merged**